spec/amp-versioning-policy.md

```diff
@@ -1,61 +1,69 @@
-# AMP versioning policy
-
-## Basics
-
-- We use [semver](http://semver.org/) terminology, but see below for changes from simple semver.
-- AMP and its extensions are versioned. There is also an **AMP release** that corresponds to the precise code. Each major and minor version may correspond to many AMP releases.
-- The AMP JS library is distributed via its major version only. The major version is expected to change rarely if ever (on the order of less than once per year).
-- Each AMP extension is versioned independently. Publishers of AMP documents can choose the major and minor version of an extension. They cannot choose the patch version.
-- AMP is **released** about once per weak and each release has a unique version number. With each release, AMP pages are rendered with the latest code for the respective major/minor versions of the AMP JS library and extensions.
+# AMP versioning and deprecations policy
 
 ## Goals
 
 - Reduce overall JS URL entropy. AMP aims to provide as few versions of the JS library and extensions as possible to improve cache hit rates for these requests.
 - Avoid breaking existing pages.
-- Keep all users of AMP on secure versions of the library.
+- Keep all users of AMP on secure and updated versions of the library.
+- Enable all AMP developers to keep their AMP pages up-to-date.
+
+## Versioning
+
+- The AMP JS library and its extensions are versioned. There is also an **[AMP release](https://github.com/ampproject/amphtml/releases/)** that corresponds to the precise code. Each major and minor version may correspond to many AMP releases.
+- AMP is **released** about once per week and each release has a unique version number. With each release, AMP pages are rendered with the latest code for the respective major/minor versions of the AMP JS library and extensions.
+
+### AMP JS library
+
+The AMP JS library is distributed via its major version only. The major version is expected to change rarely if ever (on the order of less than once per year).
+
+### AMP extensions
 
-## Non-breaking changes
+Each AMP extension is versioned independently. Publishers of AMP documents can choose the major and minor version of an extension; they cannot choose the patch version. 
 
-- If a change is backward compatible, no version should be changed.
-- This means that, for now, there is no known case under which the minor version of an extension would be changed.
+- Minor versions may be changed for breaking behavior changes *without* breaking API changes.
+- Major versions *must* be changed for any breaking API changes.
 
 ## Breaking changes
 
-Breaking changes are:
+Breaking changes must be implemented in a new version. Breaking changes are defined as:
```

Comment:
new version of amp or new version of extension?

How would this rule apply to a change to validation of non-extension html? For a concrete example, I'm trying to understand how these rules would apply to changing the syntax of the AMP boilerplate css.

Reply:
> new version of amp or new version of extension?

Made this clearer.

> How would this rule apply to a change to validation of non-extension html?

I think boilerplate CSS would fall under the new validation breaking change line item.